packages/nightingale-linegraph-track/README.md

```diff
@@ -30,7 +30,10 @@ The data expects the following structure.
 {
     name: String,
     range:[min, max],
-    colour?: Any colour,
+    colour?: Line colour,
+        (Colour will be assigned if not provided. Use "none" for no line colour)
+    fill?: Produce a filled area chart (default false),
```

Comment:
For the sake of simplicity I was wondering if we could remove the `fill` attribute and use `fillColor` as the flag instead (see suggestion below)? This would mean not having a default `fillColor`.

Reply:
yeah that also crossed my mind... so since you say that absence of default fillColor is not an issue, I'll make that change :)

Reply:
or what about just using `fill` as the single attribute? `fill: "red"` looks intuitive

Reply:
That way we get rid of the "color" vs "colour" issue!